From 91640adca6a5dd51242162ecda205938a2acd59c Mon Sep 17 00:00:00 2001
From: Tom Hochstein <tom.hochstein@nxp.com>
Date: Thu, 1 May 2025 16:49:02 -0500
Subject: [PATCH] Findtim-vx.cmake: Fix hard-coded lib dir name

The lib dir name can change for multilib case, so use the cmake variable.

Upstream-Status: Submitted
Signed-off-by: Tom Hochstein <tom.hochstein@nxp.com>
---
 cmake/modules/Findtim-vx.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/modules/Findtim-vx.cmake b/cmake/modules/Findtim-vx.cmake
index 0614dc9..3b67f5b 100644
--- a/cmake/modules/Findtim-vx.cmake
+++ b/cmake/modules/Findtim-vx.cmake
@@ -62,7 +62,7 @@ if((NOT DEFINED TIM_VX_INSTALL))
 else()
   message("=== Building with TIM_VX_LIBRIRIES from ${TIM_VX_INSTALL} ===")
   include_directories(${TIM_VX_INSTALL}/include)
-  set(LIBDIR lib)
+  set(LIBDIR ${CMAKE_INSTALL_LIBDIR})
   list(APPEND VX_DELEGATE_DEPENDENCIES ${TIM_VX_INSTALL}/${LIBDIR}/libtim-vx.so)
   if(${TIM_VX_ENABLE_NODE_TRACE})
     list(APPEND VX_DELEGATE_DEPENDENCIES ${TIM_VX_INSTALL}/${LIBDIR}/libjsoncpp.so)
-- 
2.34.1

